package.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "backend",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "lint": "eslint .",
+        "format": "prettier --log-level silent --write \"**/*.+(js|jsx|json|yml|yaml|css|md|vue|mjs|ts|tsx)\"",
+        "prepare": "husky install"
```

Comment:
Thoughts on adding a `start` command here that boots up the server using `nodemon`? So that you can just do `npm run start`